app/code/Magento/Inventory/Model/StockSourceLink/Command/GetListInterface.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Inventory\Model\StockSourceLink\Command;
+
+use Magento\Framework\Api\SearchCriteriaInterface;
+use Magento\InventoryApi\Api\Data\StockSourceLinkSearchResultsInterface;
+
+/**
+ * Find StockSourceLink list by SearchCriteria command (Service Provider Interface - SPI)
+ *
+ * Separate command interface to which Repository proxies initial GetList call, could be considered as SPI - Interfaces
+ * that you should extend and implement to customize current behaviour, but NOT expected to be used (called) in the code
+ * of business logic directly
+ *
+ * @see \Magento\InventoryApi\Api\StockSourceLinkRepositoryInterface
+ * @api
+ */
+interface GetListInterface
+{
+    /**
+     * Find StockSourceLink list by given SearchCriteria
+     * SearchCriteria is not required because load all stocks is useful case
+     *
+     * @param SearchCriteriaInterface|null $searchCriteria
+     * @return StockSourceLinkSearchResultsInterface
+     */
+    public function execute(SearchCriteriaInterface $searchCriteria = null): StockSourceLinkSearchResultsInterface;
```

Comment:
I think in this case `searchCriteria ` should be required parameter (not optional)
Because looks like we will not have case like 'load all links'